core-services/release-controller/_releases/release-ocp-4.12-ci.json

```diff
@@ -13,6 +13,7 @@
         "namespace": "origin",
         "name": "4.12",
         "excludeTags": [
+          "machine-config-operator",
```

Comment:
So this gets built by branch-ci-openshift-machine-config-operator-master-okd-images instead, I presume. Would that have to get added to per-branch (instead of just master) as well?

Reply:
yes, that image will be promoted from `master-okd-images` for now, until branching happens and master promotes to 4.12, at which point the promotion for `release-4.12-okd-images` will be enabled.
DPTP should take care of all of that when the time comes, so there should be no action required from the MCO team side.